PopUpWindow.js

```diff
@@ -0,0 +1,182 @@
+const projects = {
+  p1: {
+    name: 'Tonic     ',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'assets/Snapshoot Portfolio.png',
+    technolgies: ['HTML', 'CSS', 'Javascript'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+    generalInf: ['CANOPY', 'Back End Dev', 2023],
+  },
+  p2: {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'assets/Snapshoot Portfolio (1).png',
+    technolgies: ['HTML', 'CSS', 'Javascript'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+    generalInf: ['CANOPY', 'Back End Dev', 2023],
+  },
+  p3: {
+    name: 'Tonic     ',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'assets/Snapshoot Portfolio (2).png',
+    technolgies: ['HTML', 'CSS', 'Javascript'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+    generalInf: ['CANOPY', 'Back End Dev', 2023],
+  },
+  p4: {
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    img: 'assets/Snapshoot Portfolio (3).png',
+    technolgies: ['HTML', 'CSS', 'Javascript'],
+    linkToLiveVersion: '#',
+    linkToSource: '#',
+    generalInf: ['CANOPY', 'Back End Dev', 2023],
+  },
+};
+
+const myWork = document.querySelector('#working');
+
+function createMobileProjectCard(key) {
+  const projectSection = document.createElement('section');
+  const img = document.createElement('img');
+  const general = document.createElement('div');
+  const infGeneral = document.createElement('div');
+  const title = document.createElement('h3');
+  const inf = document.createElement('p');
+  const description = document.createElement('p');
+  const techns = document.createElement('ul');
+  const btnLoad = document.createElement('a');
+  const footer = document.createElement('footer');
+  const externalLink = document.createElement('div');
+  externalLink.appendChild(btnLoad);
+  externalLink.classList.add('external_link');
+  footer.appendChild(techns);
+  footer.appendChild(externalLink);
+  general.appendChild(title);
+  general.appendChild(infGeneral);
+  general.appendChild(description);
+  general.appendChild(footer);
+  general.classList.add('general');
+  infGeneral.appendChild(inf);
+  infGeneral.id = 'general_inf';
+  projectSection.appendChild(img);
+  img.src = projects[key].img;
+  img.alt = 'Image project';
+  img.classList.add('project_image');
```

Comment:
- [ ] You have done great to ensure that the project detail popup has a projected image. However, we should try as much as possible to ensure that we stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=Z3hCN1aRrx7pYjUK-0). Therefore because of this reason, I suggest that we ensure the image covers at least 95%+ width of the popup window. Implementing this, you should ensure the design matches the Figma design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/225566681-9f5bebd6-04ae-4b44-a437-d162c89b05d7.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/225566780-5138a5bf-c717-483d-82e7-9dd58bae7ea0.png)

Reply:
Thank you for letting me know. the issue has been rectified